unbound/datadog_checks/unbound/unbound.py

```diff
@@ -272,3 +273,10 @@ def is_exe(fpath):
                 return exe_file
 
     return None
+
+
+def hostname_to_ip(host):
```

Comment:
I'd like to see some tests and a documentation (changelog, readme, [config sample](https://github.com/DataDog/integrations-extras/blob/master/unbound/datadog_checks/unbound/data/conf.yaml.example) update for this.

Maybe "everyone" already knows this, but a comment that gethostbyname handles IP addresses as input (e.g. from https://docs.python.org/3/library/socket.html#socket.gethostbyname "If the host name is an IPv4 address itself it is returned unchanged") would help.

Don't we also need some more logic in case host doesn't contain `@`?  In that case, what is strs[1]?